dev/git.ts

```diff
@@ -200,6 +200,53 @@ const gitGenerators: Record<string, Fig.Generator> = {
       });
     },
   },
+
+  // Wildcard
+  wildcard_files_for_staging: {
+    script: "git status --short",
+    postProcess: (out) => {
+      const output = filterMessages(out);
+      if (output.startsWith("fatal:")) {
+        return [];
+      }
+
+      const items = output.split("\n").map((file) => {
+        file = file.slice(0, file.lastIndexOf("/") + 1);
+        file = file.trim();
+        const arr = file.split(" ");
+        return arr.slice(1).join(" ").trim();
+      });
```

Comment:
I would prefer not to re-assign variables. In this case, we can do something like this:
```suggestion
      const items = output.split("\n").map((file) => {
        const arr = file.slice(0, file.lastIndexOf("/" + 1))
          .trim()
          .split(" ");

        return arr.slice(1)
          .join(" ")
          .trim()
      });
```